### PR TITLE
Fixed comments and namespaces

### LINKED ISSUE #2034: Overriding Default FOSUserBundle Controllers
[This lines in the documentation](https://github.com/FriendsOfSymfony/FOSUserBundle/blame/master/Resources/doc/overriding_controllers.rst#L22-L24) cause a curious confusion:
```php
// src/AppBundle/AcmeUserBundle.php

namespace AppBundle;

//...
```
Both, the file name as the namespace suggest the reader to create a class `AcmeUserBundle.php` within the current bundle `AppBundle` even though the text is explicit:

> The first step to overriding a controller in the bundle is **to create a child bundle** whose parent is FOSUserBundle...

Maybe that lines can be change to improve the understanding of the code:

```php
// src/Acme/UserBundle/AcmeUserBundle.php

namespace Acme\UserBundle;
```
[Well as in another example (Overriding Default FOSUserBundle Templates)](https://github.com/FriendsOfSymfony/FOSUserBundle/blame/master/Resources/doc/overriding_templates.rst#L110-L122).

**If they agree I can create a PR to fix it.**

...I say **curious** because for my surprise, it was possible to do it within the same existing bundle `AppBundle`:
```php
//AppKernel.php

class AppKernel extends Kernel
{
    public function registerBundles()
    {
        $bundles = array(
            //...
            // app
            new AppBundle\AppBundle(),
            new AppBundle\UserBundle(), //\o/, although I'm not entirely sure this worked.
        );
        //...
    }
    //...
}
```

### PR DESCRIPTION
**Pull request by yceruto**
Fixed #2034

### CODE REVIEW

File: Resources/doc/overriding_controllers.rst [hunk 1]
```diff
@@ -12,9 +12,9 @@ bundle named ``AcmeUserBundle`` that declares itself a child of FOSUserBundle.
 
 .. code-block:: php
 
-    // src/AppBundle/AcmeUserBundle.php
+    // src/Acme/UserBundle/AcmeUserBundle.php
```
Comment (OskarStark) [inline Resources/doc/overriding_controllers.rst]: but `AppBundle` is SF best practice

Comment (yceruto) [inline Resources/doc/overriding_controllers.rst]: When we started a new Symfony application `AppBundle` already exists, in this case for create a child user bundle we should use a different namespace IMHO. [Reference (Overriding Default FOSUserBundle Templates)](https://github.com/FriendsOfSymfony/FOSUserBundle/blame/master/Resources/doc/overriding_templates.rst#L110-L122).

Comment (stof) [inline Resources/doc/overriding_controllers.rst]: This is indeed the case where using AppBundle does not make sense

**Comment by stof** [comment]
Thank you @yceruto.

**Pull request closed without merging**